ang/crmCaseType/timelineTable.html

```diff
@@ -76,7 +76,7 @@
           ui-jq="select2"
           ui-options="{ dropdownAutoWidth: true }"
           ng-model="activity.default_assignee_relationship"
-          ng-options="option as option.text for option in relationshipTypeOptions"
+          ng-options="option.id as option.text for option in relationshipTypeOptions"
```

Comment:
In this particular case the angular controller is creating a custom object where id is actually the relationship's label_b_a. Not 100% why it's like that, but it is how they store it in the controller and activity type. Since I'm afraid of changing it I left it.

See the definition here:
https://github.com/compucorp/civicrm-core/blob/8cdd36782d34c6d1316c63981725c480c255d7cb/ang/crmCaseType.js#L160-L162